internal/metadata/metadata.go

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright (C) 2019-2020 Vdaas.org Vald team ( kpango, rinx, kmrmt )
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Package metadata provides agent metadata structs and info.
```

Comment:
do we need this package in internal?
I think it's not general package, we can move this package to ${repo}/pkg/agent/internal/metadata or ${repo}/pkg/agent/util/metadata or ${repo}/pkg/agent/metadata

Reply:
okay. 
I think it's better to place in /pkg/agent/internal because it is referred from agent/core/ngt & agent/sidecar.